Library/Formula/objective-caml.rb

```diff
@@ -1,9 +1,23 @@
+# OCaml does not preserve binary compatibility across compiler releases,
+# so when updating it you should ensure that all dependent packages are
+# also updated by incrementing their revisions.
```

Comment:
Is there a way we can verify this in the `test do` blocks for those packages so they fail?